Fixed bug that was causing data path from config file to not be used.

benhollomon opened the pull request:
Fixed bug that was causing data path from config file to not be used.

Corrected config file path in help documentation.

IonicaBizau commented:
Can you please tell me which was the bug and how is this fixing it? Thanks! :+1:

benhollomon commented:
The path from the config file was never being used because the dataPathOpt.is_provided variable was always false. I added code to set dataPathOpt.is_provided to true when GitStats.config.path exists.

Also, the help output specified that the config file should be named '.git-stats-config.**json**', but the code was looking for a file named '.git-stats-config.**js**'. I changed the help output to match the config file referenced by the code.
